pygaggle/rerank/base.py

```diff
@@ -17,9 +17,12 @@ class Query:
     ----------
     text : str
         The query text.
+    id : Optional[str]
+        The query id.
     """
-    def __init__(self, text: str):
+    def __init__(self, text: str, id: Optional[str] = None):
```

Comment:
Do we need the `Optional` tag if there is already a default value?

Reply:
I think this is the standard convention but I might be wrong. I think that Optional[T] is the same as Union[None, T].

Reply:
I see, ok, thanks for clarifying!